src/Apps/Artwork/Components/ArtworkDetails/index.tsx

```diff
@@ -1,5 +1,8 @@
+import { ArtworkDetailsQuery } from "__generated__/ArtworkDetailsQuery.graphql"
+import { ContextConsumer } from "Artsy/Router"
 import React, { Component } from "react"
 import { createFragmentContainer, graphql } from "react-relay"
+import { QueryRenderer } from "react-relay"
 import { Box } from "Styleguide/Elements/Box"
```

Comment:
This can be imported from `@artsy/palette` (as can Flex, spacer, etc -- the layout components)